src/server/status/index.js

```diff
@@ -29,7 +34,8 @@ export default function (kbnServer, server, config) {
           build_snapshot: matchSnapshot.test(config.get('pkg.version'))
         },
         status: kbnServer.status.toJSON(),
-        metrics: kbnServer.metrics
+        metrics: kbnServer.metrics,
+        stats,
```

Comment:
So what do you think is the best way forward?  Continuing to put it on the status api and migrate as a second step, or start with a new api for these stats?

If the latter, I don't know if I like the path `_kibana/stats`.  Don't we have everything at `api/endpoint` ?  So i'd think it would be `api/stats`.  Ultimately I defer to the monitoring team, I'm not familiar enough with our api to have a firm opinion.